examples/execution-role.yaml

```diff
@@ -0,0 +1,52 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  This CloudFormation template creates a role assumed by CloudFormation
```

Comment:
just to confirm this 1 execution role can be used to deploy all 33 Atlas resources that have been published to CFN Public Registry?

Reply:
Yes, this Role can be used for all resources.